src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests/Functions/HealthChecks/HealthCheckFunctionTests.cs

```diff
@@ -75,7 +75,7 @@ public async Task WhenStrategiesReturnResults_ThenHealthCheckResponseContainsRes
         [
             new HealthCheckStrategyBuilder().WhereResultIsHealthy("Result one").Build(),
             new HealthCheckStrategyBuilder().WhereResultIsUnhealthy("Result two").Build(),
-            new HealthCheckStrategyBuilder().WhereResultIsHealthy(null).Build()
+            new HealthCheckStrategyBuilder().WhereResultIsHealthy("Result three").Build()
```

Comment:
It doesn't really matter but with this change is there any advantage of having three strategies now? Could leave it as two.

Reply:
Ah ok, I hadn't spotted that the middle one is unhealthy. Thanks!